src/Libraries/RevitNodesUI/RevitDropDown.cs

```diff
@@ -464,6 +464,55 @@ public override IEnumerable<AssociativeNode> BuildOutputAst(List<AssociativeNode
         }
     }
 
+    [NodeName("Roof Types")]
+    [NodeCategory(BuiltinNodeCategories.REVIT_SELECTION)]
+    [NodeDescription("RoofTypesDescription", typeof(Properties.Resources))]
+    [IsDesignScriptCompatible]
+    public class RoofTypes : RevitDropDownBase
+    {
+        private const string outputName = "Roof Types";
+
+        public RoofTypes() : base(outputName) { }
+
+        [JsonConstructor]
+        public RoofTypes(IEnumerable<PortModel> inPorts, IEnumerable<PortModel> outPorts) : base(outputName, inPorts, outPorts) { }
+
+        protected override SelectionState PopulateItemsCore(string currentSelection)
+        {
+            Items.Clear();
+
+            var fec = new FilteredElementCollector(DocumentManager.Instance.CurrentDBDocument);
+
+            fec.OfClass(typeof(Autodesk.Revit.DB.RoofType));
+            var elements = fec.ToElements();
```

Comment:
we're not using `fec` anywhere else, so declare `elements` above (where `fec` is now) and chain `OfClass` and `.ToElements` methods.